pkg/replication/admission.go

```diff
@@ -275,7 +275,7 @@ func (ac *AdmissionContext) validateLocationsAndTransforms(sd *comp_v1.ServiceDe
 		clusterLocations.Insert(location)
 		if sdLocation.EnvType == ac.CurrentLocation.EnvType && !ac.ReplicatedLocations.Has(location) {
 			rejectionMessages = append(rejectionMessages,
-				fmt.Sprintf("location %q does not exist in %q", sdLocation.Name, sdLocation.EnvType))
+				fmt.Sprintf("location %q (region: %q, account: %q) does not exist in %q environment, see go/micros2-locations", sdLocation.Name, sdLocation.Region, sdLocation.Account, sdLocation.EnvType))
```

Comment:
This message is not OSS-friendly =)

Reply:
The issue I had was I did not define an "account" in the SD. So, if account is required, it should say upfront "location dev does not specify an account"

Reply:
@ash2k I know, and happy to change the message once someone outside Atlassian complains about it :) for now I think it will be useful

Reply:
@cstivers78 good point, will check that as well.

Reply:
added more validation